dataflow/src/main/java/org/checkerframework/dataflow/analysis/ConditionalTransferResult.java

```diff
@@ -19,10 +21,10 @@
     private final boolean storeChanged;
 
     /** The 'then' result store. */
-    protected final S thenStore;
+    protected final Set<S> thenStore;
```

Comment:
It would be very helpful to understand the proposed changes if you added documentation to what the changed semantics is.